app/uk/gov/hmrc/gatekeeper/controllers/EmailsController.scala

```diff
@@ -43,24 +43,25 @@ import uk.gov.hmrc.gatekeeper.views.html.{ErrorTemplate, ForbiddenView}
 
 @Singleton
 class EmailsController @Inject() (
-    developerService: DeveloperService,
-    apiDefinitionService: ApiDefinitionService,
-    emailLandingView: EmailLandingView,
-    emailInformationView: EmailInformationView,
-    emailsAllUsersView: EmailAllUsersView,
-    emailApiSubscriptionsView: EmailApiSubscriptionsView,
-    emailPreferencesChoiceView: EmailPreferencesChoiceView,
-    emailPreferencesTopicView: EmailPreferencesTopicView,
-    emailPreferencesAPICategoryView: EmailPreferencesAPICategoryView,
-    emailPreferencesSpecificApiView: EmailPreferencesSpecificApiView,
-    emailPreferencesSelectApiView: EmailPreferencesSelectApiView,
-    emailPreferencesSelectTopicView: EmailPreferencesSelectTopicView,
-    val applicationService: ApplicationService,
-    val forbiddenView: ForbiddenView,
-    mcc: MessagesControllerComponents,
-    override val errorTemplate: ErrorTemplate,
-    val apmService: ApmService,
-    strideAuthorisationService: StrideAuthorisationService
+                                   developerService: DeveloperService,
```

Comment:
Please run `sbt scalafmtAll scalafixAll` as I think it will reformat to 2 space indentation.

Reply:
Done